scripts/devenv.sh

```diff
@@ -3,7 +3,14 @@
 set -eu -o pipefail
 set -x
 
-docker build --pull -t acs-engine .
+docker build --pull -t \
```

Comment:
cURL, Wget, apt-get, npm, pkgin treat empty `http_proxy` as non existent. However, custom implementations may fail if they do not account empty string in proxy variable. Safe approach would to check if http_proxy envvar is present, then call `docker build` with proxy variables. Are you OK with that approach?